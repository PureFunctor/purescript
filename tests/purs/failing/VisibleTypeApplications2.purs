-- @shouldFailWith ErrorParsingModule
module Main where

type NotAllowed @f @a = f a
