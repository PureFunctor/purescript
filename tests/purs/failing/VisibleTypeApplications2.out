Error found:
in module [33mMain[0m
at tests/purs/failing/VisibleTypeApplications2.purs:7:8 - 7:15 (line 7, column 8 - line 7, column 15)

  An expression of type:
  [33m                          [0m
  [33m  forall a. Id a => a -> a[0m
  [33m                          [0m
  cannot be applied to the type:
  [33m     [0m
  [33m  Int[0m
  [33m     [0m
  Try adding a visible type abstraction.

while inferring the type of [33mid[0m
in value declaration [33mfail[0m

See https://github.com/purescript/documentation/blob/master/errors/CannotApplyExpressionOfTypeOnType.md for more information,
or to contribute content related to this error.

